package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "litmus-ui",
   "license": "Apache-2.0",
-  "version": "1.0.5",
+  "version": "1.0.0",
```

Comment:
this needs to be upgraded...it went from 1.0.5 to 1.0.0

Reply:
will be updating to version 1.0.6